internal/testutil/assert/assertions.go

```diff
@@ -1,57 +1,33 @@
+// Copied from https://github.com/stretchr/testify/blob/master/assert/assertions.go
```

Comment:
Optional: Consider moving the `assert` package into the top-level `internal` directory instead of nesting it in `testutil`. Based on suggestion from Dave Cheney's [Practical Go](https://dave.cheney.net/practical-go/presentations/gophercon-israel.html) article, specifically section [8.6. Resist the desire to apply taxonomy](https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_resist_the_desire_to_apply_taxonomy):
> Go packages are not for creating taxonomies, `net/http` is not a type of `net` thing. Go packages are for avoid namespace collisions.

Reply:
I'd be ok moving `internal/testutil/assert` and `internal/testutil/require` to `internal/assert` and `internal/require` respectively, but there are other packages within `testutil` that seem to be applying this unhelpful "taxonomy". Should we reorganize those, too?

Reply:
We can reorganize those other `internal` packages as we modify them, but I don't think it's worth spending time to proactively move them. It's more important to get the package structure right for new or frequently changed packages.

Reply:
Sounds good. Moved `internal/testutil/assert` and `internal/testutil/require` to `internal/assert` and `internal/require` respectively.